lib/charms/layer/jenkins/plugins.py

```diff
@@ -29,16 +36,10 @@ def __init__(self):
         if proxy_hostname and proxy_port:
             if proxy_username and proxy_password:
                 proxy_address = "http://{}:{}@{}:{}".format(
-                    proxy_username,
-                    proxy_password,
-                    proxy_hostname,
-                    proxy_port
+                    proxy_username, proxy_password, proxy_hostname, proxy_port
                 )
             else:
-                proxy_address = "http://{}:{}".format(
-                    proxy_hostname,
-                    proxy_port
-                )
+                proxy_address = "http://{}:{}".format(proxy_hostname, proxy_port)
```

Comment:
f-string?

Reply:
As above, need to support Xenial which is Python 3.5 which doesn't support f-strings